nir/src/main/scala/scala/scalanative/nir/Arg.scala

```diff
@@ -0,0 +1,13 @@
+package scala.scalanative.nir
+
+case class Arg(ty: Type, passConvention: Option[PassConvention] = None)
+
+sealed abstract class PassConvention
```

Comment:
Should probably go to a separate file. We generally try to stick to one hierarchy per file.